templates/404.html

```diff
@@ -16,12 +16,24 @@
                         <h1>404 Not found</h1>
                     </div>
                     {% include "_messages.html" %}
+                    {% if languages %}
+                    <div class="alert alert-danger">
+                        {% blocktrans %}
```

Comment:
Can you add `trimmed` here, so the translation file wouldn't include `\n` chars.